dbt/adapters/snowflake/connections.py

```diff
@@ -45,6 +45,7 @@
 from dbt.events.types import AdapterEventWarning
 from dbt.ui import line_wrap_message, warning_tag
 
+from dbt.adapters.snowflake.compilation_db_replacer import CompilationDBReplacer
```

Comment:
Wouldn't it cause cyclic dependency between packages? Python will compile since it is not cyclic dependency between python files, but it will cause future architectural nightmare. Anyway, ildis suggested to create a new dependency package that will hold compilation_db_replacer.py only.